starmap/specifications/primitives/gesture_recognizers/GestureRecognizer.md

```diff
@@ -68,40 +68,20 @@ protocol GestureRecognizer {
   var enabled: Bool
 ```
 
-### States
+### State changes
 
-A gesture recognizer can be in any one of the following states:
+A gesture recognizer is notified of the following state change events:
 
 ```
-enum GestureRecognizerState {
-  case Possible
-  case Began
-  case Changed
-  case Ended
-  case Cancelled
-  case Failed
-
-  case Recognized = Ended
+protocol StateChangeObserver {
+  func didBegin(centroid: Point)
+  func didMove(centroid: Point)
```

Comment:
`centroid: Point` should be something read from the gesture recognizer, not provided to the event. Rationale being that this might not be required information by many users of this API (e.g. if you just care about rotation).